<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDeviceWidget</class>
 <widget class="QWidget" name="SettingDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="deviceIDLabel">
       <property name="text">
        <string>设备%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>网络配置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QGroupBox" name="wifiGroupBox_1">
              <property name="title">
               <string>第一个WiFi信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="tabletTracking">
                      <bool>false</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>名称：</string>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="wordWrap">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="wifiSSIDLineEdit_0"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="tabletTracking">
                      <bool>false</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>密码：</string>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="wordWrap">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="wifiPasswordLineEdit_0"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="wifiGroupBox_2">
              <property name="title">
               <string>第二个WiFi信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="tabletTracking">
                      <bool>false</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>名称：</string>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="wordWrap">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="wifiSSIDLineEdit_1"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="tabletTracking">
                      <bool>false</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>密码：</string>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="wordWrap">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="wifiPasswordLineEdit_1"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="wifiGroupBox_3">
              <property name="title">
               <string>第三个WiFi信息</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="tabletTracking">
                      <bool>false</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>名称：</string>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="wordWrap">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="wifiSSIDLineEdit_2"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="tabletTracking">
                      <bool>false</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>密码：</string>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="wordWrap">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="wifiPasswordLineEdit_2"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>MQTT配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>服务器配置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="tabletTracking">
                    <bool>false</bool>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>服务器IP地址：</string>
                   </property>
                   <property name="scaledContents">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mqttServerIpLineEdit"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="tabletTracking">
                    <bool>false</bool>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>服务器端口号：</string>
                   </property>
                   <property name="scaledContents">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mqttServerPortLineEdit"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>登录用户配置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="tabletTracking">
                    <bool>false</bool>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>MQTT用户名：</string>
                   </property>
                   <property name="scaledContents">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mqttUsernameLineEdit"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="0,0">
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="tabletTracking">
                    <bool>false</bool>
                   </property>
                   <property name="acceptDrops">
                    <bool>false</bool>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>MQTT密码：</string>
                   </property>
                   <property name="scaledContents">
                    <bool>false</bool>
                   </property>
                   <property name="wordWrap">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mqttPasswordLineEdit"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="inquireButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="returnButton">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
